From f6d5fd60f54fb9dcdc3733154637a3a214f5d5af Mon Sep 17 00:00:00 2001
From: "Eduardo Lima (Etrunko)" <etrunko@redhat.com>
Date: Thu, 1 Sep 2022 12:43:49 -0300
Subject: [PATCH] tests: Disable pci_virtio_vga for ppc64

starting QEMU: exec ./qemu-system-ppc64 -qtest unix:/tmp/qtest-2378197.sock -qtest-log /dev/null -chardev socket,path=/tmp/qtest-2378197.qmp,id=char0 -mon chardev=char0,mode=control -display none -vga none -device virtio-vga -accel qtest
stderr:
qemu-system-ppc64: -device virtio-vga: 'virtio-vga' is not a valid device model name
Broken pipe
../tests/qtest/libqtest.c:156: kill_qemu() tried to terminate QEMU process but encountered exit status 1 (expected 0)

Signed-off-by: Eduardo Lima (Etrunko) <etrunko@redhat.com>
---
 tests/qtest/display-vga-test.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/qtest/display-vga-test.c b/tests/qtest/display-vga-test.c
index ace3bb28e0..628dad4cf2 100644
--- a/tests/qtest/display-vga-test.c
+++ b/tests/qtest/display-vga-test.c
@@ -61,7 +61,7 @@ int main(int argc, char **argv)
     qtest_add_func("/display/pci/multihead", pci_multihead);
     qtest_add_func("/display/pci/virtio-gpu", pci_virtio_gpu);
     if (g_str_equal(arch, "i386") || g_str_equal(arch, "x86_64") ||
-        g_str_equal(arch, "hppa") || g_str_equal(arch, "ppc64")) {
+        g_str_equal(arch, "hppa")) {
         qtest_add_func("/display/pci/virtio-vga", pci_virtio_vga);
     }
 
-- 
2.37.2

